src/main/java/com/google/sps/data/Candidate.java

```diff
@@ -60,10 +60,32 @@ public static Builder builder() {
 
   // Creates a new Candidate object by extracting the properties from "candidateData"
   public static Candidate fromJSONObject(JSONObject candidateData) throws JSONException {
```

Comment:
Does this function still throw a JSONException after you've try-catched everything?

Reply:
Remove "throws JSONException" if not.